compiler/rustc_metadata/src/creader.rs

```diff
@@ -359,7 +359,11 @@ impl<'a> CrateLoader<'a> {
         for (_, other) in self.cstore.iter_crate_data() {
             // Same stable crate id but different SVH
             if other.stable_crate_id() == root.stable_crate_id() && other.hash() != root.hash() {
-                return Err(CrateError::SymbolConflictsOthers(root.name()));
+                span_bug!(
+                    DUMMY_SP, //FIXME:  pass in the correct span here
```

Comment:
Any ideas for how to pass in the correct span here?

Reply:
actually this could probably just be a `bug!`, there's no span associated with it.

Reply:
Thanks -- I swapped in a `bug!()` and that worked.